projects/cashmere/src/lib/stepper/stepper.component.scss

```diff
@@ -0,0 +1,102 @@
+@import '../sass/colors';
+@import '../sass/functions';
+@import '../sass/stepper.scss';
+
+.hc-stepper-green {
+    .hc-stepper-arrow {
+        @include arrow-theme($green, lighten(tint($green, 13%), 3%), $slate-gray-100, $slate-gray-200, $white, $white);
+    }
+    .hc-stepper-isolated {
+        @include isolated-theme($green, $slate-gray-200, $white, $text, $isolated-disk-size);
+    }
+}
+
+.hc-stepper-blue {
+    .hc-stepper-arrow {
+        @include arrow-theme($blue, lighten(tint($blue, 20%), 3%), $slate-gray-100, $slate-gray-200, $white, $white);
+    }
+    .hc-stepper-isolated {
+        @include isolated-theme($blue, $slate-gray-200, $white, $text, $isolated-disk-size);
+    }
+}
+
+.hc-stepper-purple {
+    .hc-stepper-arrow {
```

Comment:
The pattern I've been following so far is:

```scss
.hc-stepper-arrow {
...
  .hc-stepper-purple & { ... }
  .hc-stepper-blue & { ... }
...
}
```

Since `.hc-stepper-arrow` is the element being styled, the above pattern keeps all of its potential style variations in one place, making it easier to read IMO.

Reply:
Yup - I agree that would better.

Reply:
Fixed in the most recent commit.